### PR TITLE
Bump sbt and plugin versions

### PR DESCRIPTION
**Pull request by cb372**
Just taking care of some low hanging fruit in preparation for the Play upgrade

### CODE REVIEW

**Comment by cb372** [comment]
@sihil @philwills

File: riff-raff/build.sbt [hunk 1]
```diff
@@ -27,6 +27,9 @@ libraryDependencies ++= Seq(
   ws,
   "org.scalatestplus" %% "play" % "1.2.0" % "test",
   "com.typesafe.akka" %% "akka-testkit" % "2.3.8" % "test"
+).map((m: ModuleID) =>
+  // don't even ask why I need to do this
```
Comment (alexduf) [inline riff-raff/build.sbt]: why? :D

Comment (alexduf) [inline riff-raff/build.sbt]: (I'm not really asking, don't worry)

**Comment by philwills** [comment]
👍 , cheers @cb372

**Pull request merged**